pkg/chaosdaemon/time_server_test.go

```diff
@@ -15,93 +15,79 @@
 
 package chaosdaemon
 
-import (
-	"context"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-	"github.com/pkg/errors"
-
-	"github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/crclients"
-	"github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/crclients/test"
-	pb "github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb"
-	"github.com/chaos-mesh/chaos-mesh/pkg/log"
-	"github.com/chaos-mesh/chaos-mesh/pkg/mock"
-)
-
-var _ = Describe("time server", func() {
-	defer mock.With("MockContainerdClient", &test.MockClient{})()
-	logger, err := log.NewDefaultZapLogger()
-	Expect(err).To(BeNil())
-	s, _ := newDaemonServer(crclients.ContainerRuntimeContainerd, nil, logger)
-
-	Context("SetTimeOffset", func() {
-		It("should work", func() {
-			// Inject nil error to ignore any error
-			const ignore = true
-			defer mock.With("ModifyTimeError", ignore)()
-
-			_, err := s.SetTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).To(BeNil())
-		})
-
-		It("should fail on get pid", func() {
-			const errorStr = "mock error on load container"
-			defer mock.With("LoadContainerError", errors.New(errorStr))()
-
-			_, err := s.SetTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).ToNot(BeNil())
-			Expect(err.Error()).To(Equal(errorStr))
-		})
-
-		It("should fail on modify time", func() {
-			const errorStr = "mock error on modify time"
-			defer mock.With("ModifyTimeError", errors.New(errorStr))()
-
-			_, err := s.SetTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).ToNot(BeNil())
-			Expect(err.Error()).To(Equal(errorStr))
-		})
-	})
-
-	Context("RecoverTimeOffset", func() {
-		It("should work", func() {
-			// Inject nil error to ignore any error
-			const ignore = true
-			defer mock.With("ModifyTimeError", ignore)()
-
-			_, err := s.RecoverTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).To(BeNil())
-		})
-
-		It("should fail on get pid", func() {
-			const errorStr = "mock error on load container"
-			defer mock.With("LoadContainerError", errors.New(errorStr))()
-
-			_, err := s.RecoverTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).ToNot(BeNil())
-			Expect(err.Error()).To(Equal(errorStr))
-		})
-
-		It("should fail on modify time", func() {
-			const errorStr = "mock error on modify time"
-			defer mock.With("ModifyTimeError", errors.New(errorStr))()
-
-			_, err := s.RecoverTimeOffset(context.TODO(), &pb.TimeRequest{
-				ContainerId: "containerd://container-id",
-			})
-			Expect(err).ToNot(BeNil())
-			Expect(err.Error()).To(Equal(errorStr))
-		})
-	})
-})
+//var _ = Describe("time server", func() {
```

Comment:
We need new tests instead of ignoring them directly!

Reply:
Actually , I want to delete all the test here ,because these test is weird. The test in e2e-test & unit test in pkg/time will cover these test.